sdk/storage/storage-datalake/package.json

```diff
@@ -2,7 +2,12 @@
   "name": "@azure/storage-datalake",
   "author": "Microsoft Corporation",
   "description": "DataLakeStorageClient Library with typescript type definitions for node.js and browser.",
-  "version": "0.1.1",
+  "version": "1.0.0",
```

Comment:
Let's not update the major version from 0 to 1 just yet.
Use 0.2.0 for now

Reply:
Done